frontend/vite.config.ts

```diff
@@ -51,6 +52,17 @@ export default defineConfig({
     nodePolyfills({
       include: ['process', 'buffer', 'stream'],
     }),
+    // Make sure we copy the minified monaco-editor source into the static folder
+    // since it's loaded dynamically and not bundled via ESM. We do it this way
+    // to support setting the localization language
+    viteStaticCopy({
+      targets: [
+        {
+          src: "node_modules/monaco-editor/min/vs",
```

Comment:
there's possibly some files we can exclude from this path so we don't have to bundle them all - if we're happy with this approach then we should figure out which files are required and only include those (workers files, language files, etc.)

Reply:
Yeah absolutely - I guess the best solution might be using esm modules, with Vite chunk splitting, but to get i18n working with that approach is a lot more work, as it doesn't seem to be supported by default.

Here's a related issue for this: https://github.com/microsoft/monaco-editor/issues/1322

It's just more vite plugins though and more copying files; wrapping all the workers, etc.
Can ofcourse be done, but I felt this was a much lighter approach for now.